Update mendix-feedback.md

mel-chiu opened the pull request:
Updated the Mendix Feedback document. The images are not ready.

mel-chiu committed:
Update mendix-feedback.md

Updated the Mendix Feedback document. The images are not ready.

mel-chiu committed:
Updated images file names

Updated images file names

mel-chiu committed:
Update mendix-feedback.md

Removed emoji

mel-chiu committed:
Create mendix-feedback

Created a new image folder in Modules

mel-chiu committed:
Delete mendix-feedback

mel-chiu committed:
Add files via upload

Added mendix-feedback module images

mel-chiu committed:
Update mendix-feedback.md

Updated Image path

mel-chiu committed:
Create mendix-feedback.md in the modules folder

Create mendix-feedback.md in the modules folder

mel-chiu committed:
Delete mendix-feedback.md from the widgets folder

Delete mendix-feedback.md from the widgets folder

mel-chiu committed:
Update mendix-feedback.md

Correct the path

mel-chiu committed:
Update mendix-feedback.md

Updated the category

mel-chiu committed:
Merge branch 'mendix:development' into patch-4

ConnorLand commented:
Planned module release date: Dec 8

mel-chiu committed:
Merge branch 'mendix:development' into patch-4

ConnorLand commented:
> Planned module release date: Dec 8

Images are correct. This PR will merge later today.

ConnorLand committed:
Add alias

ConnorLand committed:
formet

ConnorLand committed:
Fix links

ConnorLand committed:
Change instances of Widget to Module

ConnorLand committed:
Change mentions of old widget versions

ConnorLand commented on content/en/docs/appstore/modules/mendix-feedback.md:
```diff
@@ -0,0 +1,158 @@
+
+### 3.3 Upgrading the Widget 
+
+Simply download the module or widget from the Marketplace. It will replace the old feedback widget in your folder. You can encounter the following error:
```
Module or widget? Shouldn't it be "Download the newest version of the Mendix Feedback Module", since that is the thing they download?

mel-chiu replied:
I changed it to Mendix Feedback Module

mel-chiu commented on content/en/docs/appstore/modules/mendix-feedback.md:
```diff
@@ -0,0 +1,158 @@
+
+The feedback provided via a widget version lower than 8.4.0 is no longer sent to the [Feedback](/developerportal/collaborate/feedback/) section of the Developer Portal, but your app will keep working as usual.
+
+If you are using the [Native Mobile Resources](/appstore/modules/native-mobile-resources/) module (which also includes a version of the Mendix Feedback widget), upgrade to version 3.5.1 or higher.
```
@ConnorLand We don't own this module. 
---- It can be changed to ---> 
If you are using the [Native Mobile Resources](/appstore/modules/native-mobile-resources/) module (which includes another version of the Mendix Native Feedback widget), upgrade to version 3.5.1 or higher.

mel-chiu committed:
Update mendix-feedback.md

Edit the description of the native mobile resources.
Added another instruction in the paragraph about upgrading the widget to the module.

mel-chiu committed:
Update mendix-feedback.md

Edited the upgrading section

mel-chiu committed:
Update mendix-feedback.md

Added version support limitation.

mel-chiu committed:
Update mendix-feedback.md

Added Atlas UI Core as a required module

mel-chiu committed:
Update mendix-feedback.md

Changed the name Atlas UI Core to Atlas Core

mel-chiu committed:
Update mendix-feedback.md

Added a reminder to configure the module role

ConnorLand committed:
Update with legacy section

mel-chiu committed:
Update mendix-feedback.md

Edited the legacy documentation section.

ConnorLand committed:
change language

mel-chiu committed:
Update mendix-feedback.md

Changed the support version of the old widget

ConnorLand committed:
Trim and border images

ConnorLand committed:
Rename old anchor links

ConnorLand committed:
Fix typo
